core/src/opcode.cpp

```diff
@@ -254,6 +258,7 @@ MemoryAccess get_memory_access(const Family family) {
     case Family::DEY:
     case Family::DEX:
     case Family::EOR:
+    case Family::ROL:
```

Comment:
Should be moved to ReadWrite

Reply:
Ok, need to do my homework a bit more...